integration/setup/setup.go

```diff
@@ -261,10 +261,15 @@ func setupCollection(tb testing.TB, ctx context.Context, client *mongo.Client, o
 	collection := database.Collection(collectionName)
 
 	// drop remnants of the previous failed run
-	_ = collection.Drop(ctx)
 	if ownDatabase {
-		_ = database.RunCommand(ctx, bson.D{{"dropAllUsersFromDatabase", 1}})
-		_ = database.Drop(ctx)
```

Comment:
`dropAllUsersFromDatabase` and `dropDatabase` do not return error for deleting empty. No harm in checking errors.